src/nodejs/neo-rpc/v2.9.0/client/cli/query.js

```diff
@@ -38,11 +38,12 @@ program
   .option('-T, --Txs', 'Only return an array of transactions for the block')
   .option('-H, --Human', 'I am human so make outputs easy for human')
   .option('-N, --Net [Net]', 'Select network [net]: i.e., TestNet or MainNet', 'TestNet')
-  .option('-m, --method [method]', 'Call node with this RPC method, default \'getBlockCount\'', 'getblockcount')
+  .option('-m, --method [method]', 'Call node with this RPC method, default \'getblockcount\'', 'getblockcount')
   .option('-p, --params [params]', 'Call RPC method with these params, default is blank', '')
   .on('--help', function(){
-    print("Note: Currently, arguments that modify the results of an RPC call are NOT IMPLEMENTED.")
-    print("      It Is highly recommended to use neo-rpc/client/cli/getNodesByX to find a node to use and then use this programs --node or -n option")
+    print('Note: Currently, arguments that modify the results of an RPC call are NOT IMPLEMENTED.')
```

Comment:
You might want to add a TODO here

Reply:
There already is one at the top of the file.